### DISCUSSION TITLE
Can I run the model training without the UI displayed on the terminal?

### DISCUSSION THREAD

**Discussion by nagacharan-tangirala** [tracel-ai/burn#2005] [Q&A] [answered]
Thanks for the fantastic work on this framework! I am trying to study a federated learning scenario using Burn. Since there will be multiple training instances, I don't want the Burn UI to be displayed. Is there an option to disable that?

**Comment by laggui** [comment, marked as answer]
You can use a custom renderer instead of the default terminal UI renderer.

Check out the [custom-renderer example](https://github.com/tracel-ai/burn/tree/main/examples/custom-renderer), it implements a very simple `MetricsRenderer` that simply uses the `dbg!` macro to print the values. You can adjust to your use case to log/display the as you wish.

**Reply by nagacharan-tangirala** [reply]
That worked as expected. Thanks a lot for answering. :)
